stream_test.go

```diff
@@ -52,6 +53,18 @@ var _ = Describe("Stream", func() {
 		resetCalledAtOffset = offset
 	}
 
+	// streamWithTimeout creates an io.ReadWriter that makes sure that Read and Write time out
+	streamWithTimeout := func(str io.ReadWriter) io.ReadWriter {
+		timeout := scaleDuration(250 * time.Millisecond)
```

Comment:
The deadline tests will fail on the CIs, because the deadline will be after the timeout.